integration_tests/e2e/overview/educationAndTrainingTab.cy.ts

```diff
@@ -78,7 +118,7 @@ context('Prisoner Overview page - Education And Training tab', () => {
       educationAndTrainingPage //
         .activeTabIs('Education and training')
         .hasFunctionalSkillsDisplayed()
-        .hasCompletedInPrisonQualificationsDisplayed()
+        .hasCompletedInPrisonQualificationsLast12MonthsDisplayed()
```

Comment:
This is definitely wrong - if this test is "the API call to learner education returned a 404" .... how does it manage to display the table that describes the prisoners completed courses in the last 12 months?

Reply:
I've only updated the function name here to be more accurate description as to what its looking for on the page; so the test must have always been wrong. I can look at rewriting the tests to work.